### PR TITLE
add federator-rest pod deployment stuff

### PR DESCRIPTION
**Pull request by YungTsun**
Deployment for Federatorai RESTFul API pod

### CODE REVIEW

File: pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go [hunk 1]
```diff
@@ -433,7 +452,7 @@ func (asp *AlamedaServiceParamter) GetUninstallPersistentVolumeClaimSource() *Re
 	pvc = sectionUninstallPersistentVolumeClaimSource(pvc, asp.AlamedaFedemeterSectionSet.Storages, "PersistentVolumeClaim/fedemeter-data.yaml", v1alpha1.Data)
 	pvc = sectionUninstallPersistentVolumeClaimSource(pvc, asp.AlamedaNotifierSectionSet.Storages, "PersistentVolumeClaim/alameda-notifier-data.yaml", v1alpha1.Data)
 	pvc = sectionUninstallPersistentVolumeClaimSource(pvc, asp.FederatoraiAgentSectionSet.Storages, "PersistentVolumeClaim/federatorai-agent-data.yaml", v1alpha1.Data)
-	pvc = sectionUninstallPersistentVolumeClaimSource(pvc, asp.FederatoraiAgentGPUSectionSet.Storages, "PersistentVolumeClaim/federatorai-agent-gpu-data.yaml", v1alpha1.Data)
```
Comment (rax845163) [inline pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go]: Is this line needs to be removed?

File: pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go [hunk 2]
```diff
@@ -505,7 +525,7 @@ func (asp *AlamedaServiceParamter) getInstallPersistentVolumeClaimSource() []str
 	pvc = sectioninstallPersistentVolumeClaimSource(pvc, asp.AlamedaFedemeterSectionSet.Storages, "PersistentVolumeClaim/fedemeter-data.yaml", v1alpha1.Data)
 	pvc = sectioninstallPersistentVolumeClaimSource(pvc, asp.AlamedaNotifierSectionSet.Storages, "PersistentVolumeClaim/alameda-notifier-data.yaml", v1alpha1.Data)
 	pvc = sectioninstallPersistentVolumeClaimSource(pvc, asp.FederatoraiAgentSectionSet.Storages, "PersistentVolumeClaim/federatorai-agent-data.yaml", v1alpha1.Data)
-	pvc = sectioninstallPersistentVolumeClaimSource(pvc, asp.FederatoraiAgentGPUSectionSet.Storages, "PersistentVolumeClaim/federatorai-agent-gpu-data.yaml", v1alpha1.Data)
```
Comment (rax845163) [inline pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go]: Is this line needs to be removed?

File: pkg/processcrdspec/globalsectionset/globalsectionset.go [hunk 3]
```diff
@@ -115,6 +115,10 @@ func GlobalSectionSetParamterToDeployment(dep *appsv1.Deployment, asp *alamedase
 		util.SetImageStruct(dep, asp.Version, util.FederatoraiAgentGPUCTN)
 		util.SetStorageToVolumeSource(dep, asp.Storages, "federatorai-agent-gpu-type.pvc", util.AlamedaGroup)
 		util.SetStorageToMountPath(dep, asp.Storages, util.FederatoraiAgentGPUCTN, "federatorai-agent-gpu-type-storage", util.AlamedaGroup)
+	case util.FederatoraiRestDPN:
+		util.SetImageStruct(dep, asp.Version, util.FederatoraiRestCTN)
+		util.SetStorageToVolumeSource(dep, asp.Storages, "federatorai-rest-type.pvc", util.AlamedaGroup)
+		util.SetStorageToMountPath(dep, asp.Storages, util.FederatoraiRestCTN, "federatorai-rest-type-storage", util.RestAPIGroup)
```
Comment (rax845163) [inline pkg/processcrdspec/globalsectionset/globalsectionset.go]: Maybe there is no needs to use the new defined group "util.RestAPIGroup".

File: pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go (hunk 1 above)
Comment (YungTsun) [reply pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go]: Sorry, it's a typo and will add it back

File: pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go (hunk 2 above)
Comment (YungTsun) [reply pkg/processcrdspec/alamedaserviceparamter/alamedaserviceparamter.go]: Sorry, it's a typo and will add it back

File: pkg/processcrdspec/globalsectionset/globalsectionset.go (hunk 3 above)
Comment (YungTsun) [reply pkg/processcrdspec/globalsectionset/globalsectionset.go]: Yes, it looks no need to add this group, will remove it

**Comment by YungTsun** [comment]
Done, please help to review it. Thanks.

**Pull request merged**